src/app/app.scss

```diff
@@ -0,0 +1,200 @@
+@import 'src/sass/mixins/index';
+@import 'src/sass/variables/index';
+
+.App {
+  display: block;
+  &-main {
+    width: 100%;
+    background-color: $primary-color;
+    display: flex;
+    flex-wrap: nowrap;
+    @media screen and (max-width: 1216px) {
+      flex-wrap: wrap;
+    }
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    &-left,
+    &-right {
+      margin: 2rem;
+    }
+  }
+  &-title {
+    font-size: 3rem;
+    font-weight: 400;
+    letter-spacing: 1px;
+    color: white;
+    margin-bottom: 1rem;
+    text-align: left;
+    @media screen and (max-width: 1216px) {
+      text-align: center;
+    }
+  }
+  &-description {
+    color: white;
+    font-weight: 400;
+    font-size: 1.15rem;
+    max-width: 50ch;
+    text-align: left;
+    @media screen and (max-width: 1216px) {
```

Comment:
Prefer variable / mixin for reused breakpoints

Reply:
I added these in last minute, I'll clean them up and move them into breakpoint variables